front-end/README.md

```diff
@@ -6,6 +6,10 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
+### `npm install`
+
+Installs npm, allowing the following commands.
```

Comment:
My only issue with what you've written is that `npm install` doesn't install npm so much as it installs the dependencies required by `package-lock.json` so that when `npm start` is run, you don't get 'can't find this module!' errors. 

I'd suggest changing this to reflect that or removing this line entirely.

Other than that, it looks good!

Reply:
Changed.